ui.apps/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/composite-multifield/source/touchui-composite-multifield.js

```diff
@@ -189,6 +189,10 @@
 
                         _.each(record, function (rValue, rKey) {
                             $field = $($fieldSets[i]).find("[name='./" + rKey + "']").last();
+																												
+																												if ($field.hasClass("coral-RichText-editable")) {
+	                               $field = $($fieldSets[i]).find("input[name='./" + rKey + "']").first();
```

Comment:
Is this not overwriting the value set on line 191?  What is the rationale for the first vs the last?

@schoudry is this a safe change?

Reply:
i think its not overwriting... indentation is so poor,` if ($field.hasClass("coral-RichText-editable")) {` is not visible

Reply:
omg....